.github/workflows/template_release_drafter.yml

```
@@ -24,6 +24,16 @@ jobs:
     runs-on: ubuntu-22.04
 
     steps:
+      - name: Create Drafter Configuration
+        if: ${{ hashFiles('.github/release-drafter.yml') != '' }}
+        run: |
+          mkdir .github
```

```suggestion
          mkdir -p .github
```
Avoid breaking the script because of existing directory